bin/requirements-frozen.txt

```diff
@@ -5,6 +5,7 @@ certifi==2023.11.17
 charset-normalizer==3.3.2
 click==8.1.7
 Flask==3.0.0
+Flask-HTTPAuth==4.8.0
```

Comment:
I prefer not to add this dependency (even if it's only a dev dependency) and make a simple check in `netrc.py`(see the proposed code next)